src/registrar/admin.py

```diff
@@ -15,6 +15,7 @@
 from django.urls import reverse
```

Comment:
[Leaving this as a code comment to thread the conversation]

@rachidatecs This update looks great! Quick question, did you include the show/hide toggle because I didn't see it. Or will that be added later in a separate ticket? 

I noticed the blue heading bars use didn't cases. They should all use sentence case, but one has all caps (CONTACT) and one has title case (Personal Info). Can you update those as part of the PR?

Reply:
Separate ticket.
Done.

Reply:
ok! Just reviewed again and looks like the "i" in the heading bar for "Personal Info" is still capitalized - should be lower case. Everything else looks good, so I'm going to approve!

Reply:
Easy fix. I'll merge it before deploying.

![Screenshot 2024-06-07 at 1 20 40 AM](https://github.com/cisagov/manage.get.gov/assets/107004823/19ac436f-63a5-45c8-8130-e1b0946bb27b)